pkgs/by-name/ir/irmago/package.nix

```
@@ -0,0 +1,47 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  lib,
+}:
+
+buildGoModule rec {
+  pname = "irmago";
+  version = "0.18.1";
+
+  src = fetchFromGitHub {
+    owner = "privacybydesign";
+    repo = "irmago";
+    tag = "v${version}";
+    hash = "sha256-hHMwyPlPdPvznH8I4LmIv7S4jvzgleDuUPrBq+aeIHA=";
+  };
+
+  vendorHash = "sha256-A9HGxrJjtFERKoKv6/mHuL5m5IX29xECt3Jlv5nry3k=";
+
+  subPackages = [ "." ];
```

the upstream suggests building `irma` subdir, not the root
```suggestion
  subPackages = [ "irma" ];
```
https://github.com/privacybydesign/irmago#using-go and .github/actions/build/action.yml